docs/ldml/tr35-keyboards.md

```diff
@@ -308,6 +308,77 @@ For extensibility, the `<special>` element will be allowed at nearly every level
 
 See [Element special](tr35.md#special) in Part 1.
 
+## Normalization
+
+Normalization will not typically be the responsibility of the keyboard author, rather this will be managed by the implementation.
+The implementation will apply normalization as appropriate when matching transform rules and `<display>` value matching.
+Output from the keyboard, following application of all transform rules, will be normalized to implementation- or application-requested form.
+
+The attribute value `normalization="disabled"` can be used to indicate that no automatic normalization happens.
+Using this setting is not recommended. See [`<settings>`](#element-settings) for details.
+
+Input source files may be in any normalization format, however, authors should be aware of two areas where normalization affects keyboard operation: that of transform matching, and that of output.
+
+Normalization can be applied within segments defined by Grapheme Cluster Boundaries as defined by [UAX #29](https://www.unicode.org/reports/tr29/#Grapheme_Cluster_Boundaries). This segmentation is pertinent where markers are involved, see below.
+
+### Normalization and Transform Matching
+
+Regardless of the normalization form in the keyboard source file or in the edit buffer context, transform matching will be performed using **NFD**. For example, all of the following transforms will match the input strings `è̠`, whether the input is U+00E8 U+0320, U+0065 U+0320 U+0300, or U+0065 U+0300 U+0320.
+
+```xml
+<transform from="e\u{0300}\u{0320}" /> <!-- Non-NFD but will match -->
+<transform from="e\u{0320}\u{0300}" />
+<transform from="\u{00E8}\u{0320}" /> <!-- è + U+0320 -->
```

Comment:
```suggestion
<transform from="e\u{0320}\u{0300}" /> <!-- NFD -->
<transform from="e\u{0300}\u{0320}" /> <!-- Non-NFD but will match -->
<transform from="\u{00E8}\u{0320}" /> <!-- è + U+0320 -->
```

Put the NFD transform first for clarity I think

Reply:
did this